gravitee-am-ui/src/app/domain/settings/theme/help-tips/help-tips.component.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {Component, Input, OnInit, SimpleChanges} from '@angular/core';
+
+@Component({
+  selector: 'app-help-tips',
+  templateUrl: './help-tips.component.html',
+  styleUrls: ['./help-tips.component.scss']
+})
+export class HelpTipsThemeComponent implements OnInit {
```

Comment:
Question: do we need to implements `OnChanges` as `ngOnChanges` is overridden on line 35?

Reply:
FP